sw/airborne/subsystems/abi_sender_ids.h

```diff
@@ -362,4 +362,13 @@
 #define RANGE_FORCEFIELD_ID 1
 #endif
 
+/*
+ * ID's for computer vision systems
```

Comment:
Not sure if this an appropriate category given the other abi naming. There are already some other computer vision abi messages here but are named based on their message type not the sensor. I have no problem with naming the sensor rather than the use but then the other ids should be updated. You can also just make an issue of it to be updated later.

Reply:
changed it to `CAM_JEVOIS_ID`